documentation/src/main/doc/modules/connectors/partials/META-INF/connector/smallrye-kafka-incoming.adoc

```diff
@@ -133,6 +133,10 @@ Type: _string_ | false |
 
 Type: _string_ | false | 
 
+| *fail-on-deserialization-failure* | When no deserialization failure handler is set and a deserialization failure happens, report the failure and mark the application as unhealthy. If set to `false`, the record is skipped.
+
```

Comment:
> the record is skipped.

I feel this is a little confused,  actually consumer still gets a null record. maybe rephrase like "a fallback value `null` is returned to Kafka consumer".

Reply:
Fixed.